app/views/fake/show.html.erb

```diff
@@ -36,7 +36,7 @@
 
           <div class="col-xl-6 mb-4">
             <div class="form-floating">
-              <%= form.select :itunes_type, [["Full", "full"], ["Bonus", "bonus"], ["Invalid", "invalid"]], include_blank: true %>
```

Comment:
I needed to test this because there was some weird arrow appearing in the background. Needed to ensure it wasn't part of the actual select option. (It wasn't!)